new file changes (#267)
SeleniumHQ/seleniumhq.github.io: docs_source_files/content/getting_started/quick.en.md

## WebDriver

> If you are beginning with desktop website or mobile website test automation, then you

Reviewer: ```suggestion
If you are beginning with website test automation, then you
```

Author: @diemol  
I run Selenium Test cases for both the mobile site and desktop website inside the chrome browser, so I wanted this text to be more generic.

Reviewer: Exactly, that's why we can remove `desktop` and just say `websites`, which should cover everything, don't you think?

Author: @diemol  
okay, but I have seen lots of Automation testers that they don't know the fact that they too can automate the mobile websites inside the chrome browser when their app uses the common web views.

This could be helpful to the entire community.

Reviewer: Fair enough, let's leave it as you proposed, thanks!

Author: @diemol  Thanks

> If you are beginning with desktop website and mobile website inside the chrome browser for the test automation, then you

Reviewer 2: ```suggestion
If you are beginning with desktop website or mobile website test automation, then you
```
